src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -227,25 +229,14 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
               </Button>
             </>
           ) : (
-            <>
-              <SnapshotBulkDelete
-                instance={instance}
-                snapshotNames={selectedNames}
-                onStart={() => setProcessingNames(selectedNames)}
-                onFinish={() => setProcessingNames([])}
-                onSuccess={onSuccess}
-                onFailure={onFailure}
-              />
-              <Button
-                appearance="link"
-                className="u-no-padding--top"
-                hasIcon
-                onClick={() => setSelectedNames([])}
-              >
-                <span>Clear selection</span>
-                <Icon name="close" className="clear-selection-icon" />
-              </Button>
-            </>
+            <SnapshotBulkDelete
+              instance={instance}
+              snapshotNames={selectedNames}
+              onStart={() => setProcessingNames(selectedNames)}
+              onFinish={() => setProcessingNames([])}
+              onSuccess={onSuccess}
+              onFailure={onFailure}
+            />
```

Comment:
Shouldn't this be right-aligned, as the other CTAs are? It's a bit confusing to find this button on the left.

![Peek 2023-10-12 11-57](https://github.com/canonical/lxd-ui/assets/56583786/e4f40814-cb44-4da2-b17c-276032980afb)

What do you think @piperdeck ?

Reply:
I think it makes a lot of sense to keep the bulk actions on the left side together with the bulk select. This is also consistent with the instance list bulk actions.

Reply:
In the instance case it's different. There are a bunch of bulk actions so it's hard to miss them. In this case, I had to look around for a while, confused, to find it. This probably needs user research to confirm, but my opinion as in "the one who hadn't visited this page for a while", so more or less "fresh eyes", is that this is not the right place to put a button, especially given that in the base case you have the buttons on the opposite side. Let's see what @piperdeck thinks about it - maybe a third way is also possible.